package.json

```diff
@@ -63,7 +63,8 @@
     "pnpm": {
         "overrides": {
             "@testing-library/jest-dom>@adobe/css-tools@<4.3.2": ">=4.3.2",
-            "nx-cloud>axios@<1.6.0": ">=1.6.0"
+            "nx-cloud>axios@<1.6.0": ">=1.6.0",
+            "@jest/environment": "29.5.0"
```

Comment:
A number of modules, `@sap-ux/deploy-tooling`, `@sap-ux/ui5-library-inquirer` & `@sap-ux/app-config-writer` because this dev dependency was updating to `29.6.4`.
Please advise if there is a better approach

Reply:
I've removed the override, I was no longer seeing the errors and the build is green.